interpreter/dotnet/instance.go

```diff
@@ -622,7 +622,7 @@ func (i *dotnetInstance) SynchronizeMappings(ebpf interpreter.EbpfHandler,
 			info.fileID, m.Path,
 			info.simpleName, info.guid)
 
-		if !info.reported {
+		if !symbolReporter.ExecutableKnown(info.fileID) {
 			open := func() (process.ReadAtCloser, error) {
 				return os.Open(m.Path)
```

Comment:
```suggestion
				return pr.OpenMappingFile(m)
```
I know this is unmodified code, but caught my eye. Would be ok to fix this while at it, or requires separate PR?